src/app/core/translations/translate-loader.service.ts

```diff
@@ -29,6 +29,23 @@ export enum LengthOfStayLabels {
   LONGER = 'LONGER',
 }
 
```

Comment:
Enumy i interface trzymamy w shared/models. consty w shared/consts
Natomiast cy to jest potrzebne? Może skorzystajmy z modeli w api/core?
src/app/core/api/model/healthOffer.ts

Reply:
Dzięki, zrobione.